source/NVDAObjects/JAB/__init__.py

```diff
@@ -97,13 +97,17 @@
 }
 
 
+re_simpleXmlTag = re.compile(r"(\<[^>]+\>)+")
 
-re_simpleXmlTag=re.compile(r"\<[^>]+\>")
 
+def _subHtmlTag(match):

```

Comment:
Can this please get type definitions and a doc string?

```suggestion
def _subHtmlTag(match: re.match) -> str:
```